src/interactionManager/authorizationFlow.ts

```diff
@@ -37,17 +37,9 @@ export class AuthorizationFlow extends Flow<
   }
 
   public async consumeAuthorizationRequest(request: AuthorizationRequest) {
-    if (!request.description) return false
-
-    if (!this.state.description.length) {
-      const { description, imageURL, action } = request
-      this.state.description = description
-      imageURL && (this.state.imageURL = imageURL)
-      action && (this.state.action = action)
-      return true
-    }
-
-    return false
+    const { callbackURL, ...responseProps } = request
```

Comment:
It's safer to actually de-structure the `request` explicitly instead of using the spread operator, as there might be other (illegal) properties on the request objects (can't really trust TypeScript....)